content/ja/docs/setup/production-environment/turnkey/aws.md

```diff
@@ -33,10 +33,9 @@ To create a Kubernetes cluster on AWS, you will need an Access Key ID and a Secr
 
 ### コマンドライン管理ツール: kubectl
 
-The cluster startup script will leave you with a `kubernetes` directory on your workstation.
-Alternately, you can download the latest Kubernetes release from [this page](https://github.com/kubernetes/kubernetes/releases).
+クラスターの起動スクリプトによってワークステーション上に`kubernetes`ディレクトリが作成されます。もしくは、Kubernetesの最新リリースを[こちら](https://github.com/kubernetes/kubernetes/releases)からダウンロードすることも可能です。
 
-Next, add the appropriate binary folder to your `PATH` to access kubectl:
+次に、kubectlにアクセスするために適切なバイナリフォルダーを`PATH`へ追加します:
 
 ```shell
 # macOS
```

Comment:
Is it helpful to localize `path/to/kubernetes-directory` on the following lines?

Reply:
@nasa9084 @sftim imho `path/to/dir` is a fairly common expression in Japanese too.

Reply:
Let's resolve this since it's not necessary